themes/90210/README.md

```diff
@@ -0,0 +1,15 @@
+# 90210 💜
+
+![alt text](90210-dark.png)
+
+- **Brown**: Used to indicate a dirty state in source control management (SCM) and Git prompts.
+- **Bold Green**: Signifies a clean state in SCM and Git prompts.
+- **Green**: Used as a prefix and suffix in SCM and Git prompts.
+- **Bold Gray**: Forms the main structure of the prompt, displaying time, user, host, and current working directory.
+- **Navy**: Represents the time in the prompt.
+- **Green (again)**: Denotes the user and host in the prompt.
+- **Olive**: Marks the separation between the username and hostname.
+- **Purple**: Indicates the current working directory in the prompt.
+- **Reset Color**: Ensures proper color reset after prompt elements.
```

Comment:
I don't think the item "Reset Color" explains something useful to the user. It's not visible to the user.